lib/source/layouts/_cookie_banner.erb

```diff
@@ -0,0 +1,108 @@
+<%
+  is_enabled ||= config.tech_docs.dig(:cookie_banner, :enable)
+  essential_cookies = config.tech_docs.dig(:cookie_banner, :cookie_types) ? config[:tech_docs][:cookie_banner][:cookie_types].include?('essential') : false
+
+  analytics_cookies = config.tech_docs.dig(:cookie_banner, :cookie_types) ? config[:tech_docs][:cookie_banner][:cookie_types].include?('analytics') : false
+
+  additional_cookies = config.tech_docs.dig(:cookie_banner, :cookie_types) ? config[:tech_docs][:cookie_banner][:cookie_types].include?('additional') : false
+
+  custom_cookies_text = config.tech_docs.dig(:cookie_banner, :custom_cookies_text)
+
+  default_essential_cookies_text = "We use some essential cookies to make this service work."
+
+  default_analytics_cookies_text = "We’d also like to use analytics cookies so we can understand how you use the service and make improvements."
+
+  default_additional_cookies_text = "We’d like to set additional cookies so we can remember your settings, understand how people use the service and make improvements."
```

Comment:
I have a feeling this could be simplified further... I had a go in a branch which you can see here: https://github.com/alphagov/tech-docs-gem/compare/add-cookie-banner...refactor-cookie-message

Let me know what you think!

Reply:
yeah makes sense and removes the need for extra variables as well

Reply:
updated, thanks